src/arrayMethodSort.js

```diff
@@ -4,8 +4,29 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
     // write code here
+    const defaultCompare = (el1, el2) => {
+      const itemOne = `${el1}`;
+      const itemTwo = `${el2}`;
+
+      return itemOne > itemTwo ? +1 : -1;
```

Comment:
The default compare function does not handle the case where two elements are equal (it should return 0 in that case). This is important because sort stability in JavaScript is guaranteed as of ES2019, so equal elements should retain their original order if they are compared.